v1.0/openapi3/schemas/Name.yaml

```diff
@@ -1,4 +1,13 @@
 title: Name
 type: string
 pattern: '^(?!\s*$)[\p{L}\p{Nd} .,''-]{1,128}$'
-description: "The API data type Name is a JSON String, restricted by a regular expression to avoid characters which are generally not used in a name.\nRegular Expression - The regular expression for restricting the Name type is \"^(?!\\s*$)[\\w .,'-]{1,128}$\". The restriction does not allow a string consisting of whitespace only, all Unicode characters are allowed, as well as the period (.) (apostrophe (‘), dash (-), comma (,) and space characters ( ).\n\n\n**Note:** In some programming languages, Unicode support must be specifically enabled. For example, if Java is used, the flag UNICODE_CHARACTER_CLASS must be enabled to allow Unicode characters."
+description: |
+  The API data type Name is a JSON String, restricted by a regular expression
+  to avoid characters which are generally not used in a name.\n
+  Regular Expression - The regular expression for restricting the
+  Name type is \"^(?!\\s*$)[\\w .,'-]{1,128}$\". The restriction does not
+  allow a string consisting of whitespace only, all Unicode characters are
+  allowed, as well as the period (.) (apostrophe (‘), dash (-), comma (,)
+  and space characters ( ).\n\n\n**Note:** In some programming languages,
```

Comment:
Is it intended to have three newline characters here?

Reply:
Yep. These definitions are basically cut straight from https://github.com/mojaloop/mojaloop-specification/blob/master/Supporting%20Files/fspiop-rest-v1.0-OpenAPI-implementation_openapi3.yaml

We can always evaluate descriptions later.